src/common/config_parser.cpp

```diff
@@ -835,6 +838,22 @@ void ConfigParser::addSuboptionsULR(cli::CLIWrapper& cli) {
   // clang-format on
 }
 
+void ConfigParser::addSuboptionsQuantization(cli::CLIWrapper& cli) {
+  // clang-format off
+  // model quantization training
+  cli.add<size_t>("--quantize-bits",
+     "Number of bits to compress model to. Set to 32 to disable",
+      32);
+  cli.add<size_t>("--quantize-optimization-steps",
+     "Adjust quantization scaling factor for N steps",
+     0);
+  cli.add<bool>("--quantize-log",
+     "Uses log-based quantization");
```

Comment:
(minor: This may be confusable with log as in log file. Is there a better name? `--quantize-log-based`? Not sure. How do folks feel about this?)

Reply:
updated